docs/views/templates/check-your-answers.html

```diff
@@ -9,159 +9,178 @@
 {% endblock %}
 
 {% block content %}
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
 
-      <h1 class="govuk-heading-xl">
-        Check your answers before sending your application
-      </h1>
+      <h1 class="govuk-heading-xl">Check your answers before sending your application</h1>
 
-      <!-- only add a heading for a list if there is more than one list -->
-      <h2 class="govuk-heading-m">
-        Personal details
-      </h2>
+      <h2 class="govuk-heading-m">Personal details</h2>
 
-      <!-- use additional modifier class to change column widths for multiple groups of questions and answers: -->
-      <!--   * `app-check-your-answers--short` for short questions -->
-      <!--   * `app-check-your-answers__questions-long` for long questions -->
-      <!--   * none for single group of q&a -->
-      <dl class="app-check-your-answers app-check-your-answers--short">
-        <div class="app-check-your-answers__contents">
-          <dt class="app-check-your-answers__question">
+      <dl class="govuk-summary-list govuk-!-margin-bottom-9">
```

Comment:
Just out of interest, what happened to `app-check-your-answers--short` and `app-check-your-answers__questions-long` in the previous approach? Do we only provide one now and if so, which one?

Reply:
The default is the equivalent of `app-check-your-answers--short` (30%) but can be used with the [width override classes](https://design-system.service.gov.uk/styles/spacing/#width-override-classes) depending on your content.

For example:
<img width="1020" alt="screen shot 2019-01-12 at 06 46 05" src="https://user-images.githubusercontent.com/23356842/51070250-29646080-1636-11e9-8e14-ec3250891796.png">

Reply:
awesome, thanks!